packages/components/src/components/table/table-head-cell/table-head-cell-styles.ts

```diff
@@ -38,28 +46,53 @@ export const getComponentCss = (
       ...(sortable
         ? {
             button: {
+              transform: 'translate3d(0,0,0)', // creates new stacking context
```

Comment:
Why would we need one? Nothing is `position: fixed` here.  
Most likely a `position: relative` would achieve the same without rendering multiple instances of this component on the GPU.

Reply:
Fixed it